posthog/event_usage.py

```diff
@@ -267,6 +268,12 @@ def report_user_action(user: User, event: str, properties: Optional[dict] = None
     )
 
 
+@sync_to_async
```

Comment:
Nitpick: as we call it from async context, would adding `thread_sensitive=True` make sense here, to avoid creating a new thread?